tests/run_all_tests.py

```diff
@@ -516,17 +561,23 @@ def main(argv=None):
                         default='run_tests',
                         help='Choose action')
     parser.add_argument('--dont_clean_output_files', action='store_true', help='Do not clean output files')
+    parser.add_argument('--check_run_time', action='store_true', help='Print tests_failed_runtime_check.csv, '
+                                                   'list of tests with unexpected run time performance')
 
     args = parser.parse_args(argv)
     test_type = args.type
     action = args.action
     clean_out_files = not args.dont_clean_output_files
+    check_run_time = args.check_run_time
     if action != 'run_tests' and test_type != 'output':
         print(f'action: {action} is not supported with test type: {test_type}')
         sys.exit(1)
+    # if check_run_time and test_type != 'general':
+    #     print(f'check_run_time flag is not supported with test type: {test_type}')
+    #     sys.exit(1)
```

Comment:
Hi,
I kept this commented because I am not sure if it is necessary since I've added the check only for general tests.
would like to know your opinion

Reply:
I think we can keep it because if someone uses this flag for unsupported test type, this would issue an error